### PR TITLE
chore(web): tests for frontpage items

### LINKED ISSUE #7622: Test Case: Featured links (under search bar) can be opened

### LINKED ISSUE #7630: Test Case: Burger menu should show service categories

### PR DESCRIPTION
**Pull request by busla**
closes #7622, closes #7630

# ...

Attach a link to issue if relevant

## What

Specify what you're trying to achieve

## Why

Specify why you need to achieve this

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

**Comment by pshomov** [comment]
Looks like a lot of changes. Maybe you are targeting the wrong branch?

File: apps/web/components/Menu/Menu.tsx [hunk 1]
```diff
@@ -40,7 +41,10 @@ export const Menu = ({
   return (
     <MenuUI
       baseId="Menu"
-      mainLinks={mainLinks}
+      mainLinks={mainLinks.map((item) => ({
+        ...item,
+        dataTestId: 'mega-menu-link',
+      }))}
```
Comment (kristofer-lp8f) [inline apps/web/components/Menu/Menu.tsx:47]: `dataTestId` will be non-unique. Maybe use `` `data-testid='mega-menu-link-${item.id}` `` or something.

Also, `dataTestId` should be `data-testid`

**Review by kristofer-lp8f — APPROVED** [review]

**Comment by pshomov** [comment]
Seems we managed to work on the same test, my bad for not putting my name in the Notion doc. Here is for what its worth - https://github.com/island-is/island.is/pull/7637. Feel free to close it and delete the branch @busla

**Comment by busla** [comment]
you mean "highlighted links"?

**Review by pshomov — APPROVED** [review]
LGTM

**Pull request merged**